### DISCUSSION TITLE
bash 配置、脚本、命令的模块化管理框架：one.bash

### DISCUSSION THREAD

**Discussion by adoyle-h** [just-talks/tech-talks#73] [3.分享创造]
bash 配置、脚本、命令的模块化管理框架：https://github.com/one-bash/one.bash

2 年前我开源的一个项目，最近改了一个大版本，项目基本稳定了。推荐 bash 用户使用。zsh 用户用不到。

**Comment by kalxd** [comment]
fish原神，launch！
